src/client/ui/dialogs/PublishDialog.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styles from "./dialog.scss";
+import Button from "../Button";
+import Header from "../Header";
+import StringInput from "../inputs/StringInput";
+
+export default class PublishDialog extends Component {
+  static propTypes = {
+    onPublish: PropTypes.func.isRequired,
+    onCancel: PropTypes.func,
+    hideDialog: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      description: ""
+    };
+  }
+
+  render() {
+    const { onPublish, onCancel, hideDialog } = this.props;
+    return (
+      <div className={styles.dialogContainer}>
+        <Header title="Publish to Hubs" />
+        <div className={styles.publishContainer}>
+          <div className={styles.content}>
+            <img className={styles.sceneThumbnail} />
+            <div>
+              <div className={styles.inputField}>
```

Comment:
Additional field I forgot to add to the db schema is "Composed By Display Name"

Reply:
probably optional (can be left blank)

Reply:
Made #220 for it.